core/tests/test_utils.py

```diff
@@ -71,11 +72,27 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Set mock content, status_code and headers for Url Service mock
```

Comment:
Sets the content, status code and headers that the urlfetch mock should return with.